.circleci/config.yml

```diff
@@ -117,15 +123,14 @@ jobs:
 
   get_data:
     machine:
-      # Ubuntu 14.04 with Docker 17.10.0-ce
-      image: ubuntu-1604:202007-01
+      image: ubuntu-2004:202104-01
     working_directory: /home/circleci/data
     steps:
       - restore_cache:
           keys:
-            - data-v8-{{ .Branch }}-{{ .Revision }}
-            - data-v8-{{ .Branch }}
-            - data-v8-
+            - data-v9-{{ .Branch }}-{{ .Revision }}
+            - data-v9-{{ .Branch }}
+            - data-v9-
```

Comment:
Why are we dropping this cache? Abundance of caution?

Reply:
Actually, now that you made me think about it, we want to keep v8 and, if ever changed, do not update to v9 (which is used by master), but by something like `data-20.2-v1` etc.

It's been long since the last time we made changes to the data, and with `nipreps-data` this cache should become more stable and manageable with datalad (rather than just circle).